ansible/roles/utilities/create-vfs/tasks/main.yaml

```diff
@@ -0,0 +1,13 @@
+---
+- include_role:
+    name: projects-common/get-mlx-interfaces
+- include_role:
+    name: projects-common/clear-vfs
+  loop: "{{ pfs }}"
+  loop_control:
+    loop_var: pf
```

Comment:
Since `clear-vfs` already is included in the `create-vfs` role, do we need to include it here?

Reply:
removed